swift-sdk/IterableNotificationMetadata.swift

```diff
@@ -8,68 +8,59 @@
 
 import Foundation
 
-/**
- `IterableNotificationMetadata` represents the metadata in an Iterable push notification
- */
-@objc public class IterableNotificationMetadata: NSObject {
```

Comment:
This class is part of public API, this change will remove `isRealCampaignNotification` from `IterablePushNotificationMetadata`.

Reply:
Disregard this, this class should actually be private as it's not returned by any of the public methods.